lib/darf_payment/darf_payment.ex

```diff
@@ -0,0 +1,300 @@
+defmodule StarkBank.DarfPayment do
+  alias __MODULE__, as: DarfPayment
+  alias StarkBank.Utils.Rest
+  alias StarkBank.Utils.Check
+  alias StarkBank.User.Project
+  alias StarkBank.User.Organization
+  alias StarkBank.Error
+  
+  @moduledoc """
+  Groups DarfPayment related functions
+  """
+  
+  @doc """
+  When you initialize a DarfPayment, the entity will not be automatically
+  created in the Stark Bank API. The 'create' function sends the objects
+  to the Stark Bank API and returns the list of created objects.
+  
+  ## Parameters (required):
+    - `:description` [string]: Text to be displayed in your statement (min. 10 characters). ex: "payment ABC"
+    - `:revenue_code` [string]: 4-digit tax code assigned by Federal Revenue. ex: "5948"
+    - `:tax_id` [tax_id]: tax id (formatted or unformatted) of the payer. ex: "12.345.678/0001-95"
+    - `:competence` [Date or string]: competence month of the service. ex: ~D[2020-03-25]
+    - `:nominal_amount` [int]: amount due in cents without fee or interest. ex: 23456 (= R$ 234.56)
+    - `:fine_amount` [int]: fixed amount due in cents for fines. ex: 234 (= R$ 2.34)
+    - `:interest_amount` [int]: amount due in cents for interest. ex: 456 (= R$ 4.56)
+    - `:due` [Date or string]: due date for payment. ex: ~D[2020-03-25]
+
+  ## Parameters (optional):
+    - `:reference_number` [string]: number assigned to the region of the tax. ex: "08.1.17.00-4"
+    - `:scheduled` [Date or string, default today]: payment scheduled date. ex: ~D[2020-03-25]
+    - `:tags` [list of strings]: list of strings for tagging
+  
+  ## Attributes (return-only):
+    - `:id` [string, default nil]: unique id returned when payment is created. ex: "5656565656565656"
+    - `:status` [string, default nil]: current payment status. ex: "success" or "failed"
+    - `:amount` [int, default nil]: amount automatically calculated from line or bar_code. ex: 23456 (= R$ 234.56)
+    - `:fee` [integer, default nil]: fee charged when the tax payment is created. ex: 200 (= R$ 2.00)
+    - `:updated` [string, default nil]: latest update datetime for the payment. ex: "2020-03-10 10:30:00.000"
```

Comment:
padrão de exemplo de datetime é com sigil, tipo - `:created` [DateTime, default nil]: creation datetime for the Boleto. ex: ~U[2020-03-26 19:32:35.418698Z]